apps/prairielearn/assets/scripts/application.ts

```diff
@@ -1,4 +1,4 @@
-import './behaviors/custom-file-input.js';
```

Comment:
This would remove the custom-file-input feature from bootstrap-4 feature flag users, is that acceptable?

Reply:
Oof... this dual compatibility is going to drive me insane. Great catch, let me fix.

Reply:
Added back in c272ea6.